src/observability/summarizer.go

```diff
@@ -73,6 +73,27 @@ func extractNetworkInfoFromSystemLog(netLog pb.Alert, pods []types.Pod, services
 	return ip, destNs, destLabel, port, bindPort, bindAddress, protocol, nwrule, nil
 }
 
+func extractSyscallInfoFromSystemLog(sysCallLog pb.Alert, pods []types.Pod, services []types.Service) (string, string, string, string, error) {
+
+	var ParentProcess, ChildProcess, Syscall, Parameters string = "", "", "", ""
```

Comment:
```suggestion
	var parentProcess, childProcess, sysCall, parameters string = "", "", "", ""
```

Reply:
@seswarrajan Thank you for review and suggestions. Variable names are modified.